src/helm/benchmark/scenarios/scenario.py

```diff
@@ -82,6 +87,10 @@ class Output:
     """
 
     text: str
+    """The text of the output."""
```

Comment:
Same comments as for `Input` - do we actually need two classes for this or could they be one?

Reply:
Do you mean for `Input` and `Output`?